meta/lunch_and_learn.md

```diff
@@ -31,10 +37,6 @@ So far, from external contributors we've had:
 
 You're probably seeing this as an external developer to Artsy, who has been invited to talk.
 
-## Format
-
-The format is definitely not set in stone, however these are what has worked for us internally and for past contributors.
-
```

Comment:
Any reason for this? This doc is mostly used for getting inbound people and letting them say they can make use of it as they want is valuable